src/cli/opts.rs

```diff
@@ -58,24 +58,24 @@ pub enum Command {
     #[clap(aliases = &["ls"])]
     ListSwaps,
 
-    /// Lists public offers created by daemon
+    /// Lists public deals created by daemon
     #[clap(aliases = &["lo"])]
-    ListOffers {
+    ListDeals {
         #[clap(
             short,
             long,
             default_value = "open",
             possible_values = &["open", "Open", "inprogress", "in_progress", "ended", "Ended", "all", "All"],
         )]
-        select: OfferSelector,
+        select: DealSelector,
     },
 
-    /// Gives information on an open offer
+    /// Gives information on an open deal
     #[clap(aliases = &["oi"])]
```

Comment:
Change to `di` aliases

Reply:
64b2f5c7fd5ce32f6b166ed1a6f8f9dfb62f2606